storage/remote/queue_manager.go

```diff
@@ -468,6 +501,19 @@ func releaseLabels(ls labels.Labels) {
 	}
 }
 
+func internMetadata(meta storage.Metadata) storage.Metadata {
+	meta.Type = textparse.MetricType(interner.intern(string(meta.Type)))
```

Comment:
this doesn't seem correct? should it not be just `interner.intern(string(meta.Type))`?